aws-batch-pdal/batch/pipeline/script.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import os
+import boto3
+import aws_helper
+
+#this script
+print('Python script: ', sys.argv[0])
+#filelist
+print('files list: ', sys.argv[1])
+#Read Role
+print('AWS Role read: ', sys.argv[2])
+#Write Role
+print('AWS Role write: ', sys.argv[3])
+#Bucket Read
+print('S3 bucket read: ', sys.argv[4])
+#Bucket Write
+print('S3 bucket write: ', sys.argv[5])
+#Bucket Write
+print('Extracted file name suffix: ', sys.argv[6])
+
+aws_read_role = sys.argv[2]
+aws_write_role = sys.argv[3]
+aws_bucket_read = sys.argv[4]
+aws_bucket_write = sys.argv[5]
+file_name_suffix = sys.argv[6]
+file_dst_folder = sys.argv[7]
+
+tmp_dir = "tmp"
+
+sts_client = boto3.client('sts')
+src_bucket = aws_helper.get_bucket(sts_client, aws_read_role, aws_bucket_read)
+
+files_list = sys.argv[1].split(';')
+
```

Comment:
This would be best done in processing pipeline, 

```python
s3a = get_s3('s3://foo')
s3b = get_s3('s3://bar')

def file_pipeline(file_name)
   s3a.get()
   process(step)
   s3b.put()

# TODO find something to help us do this
run_pipline(file_list, file_pipeline)
```